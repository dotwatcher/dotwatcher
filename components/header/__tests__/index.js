import 'jest-styled-components';

import Header from '../index';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

describe('<Header />', () => {
	it('renders <Header />', () => {
		const tree = renderer.create(<Header title="Page Title" />).toJSON()
		expect(tree).toMatchSnapshot()
	});
});
