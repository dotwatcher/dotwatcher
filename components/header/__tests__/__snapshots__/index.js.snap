// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders <Header /> 1`] = `
.c1 {
  display: table-cell;
  margin: var(--spacing-none);
  font-weight: 600;
  line-height: 1;
}

.c0 {
  background-color: var(--near-black);
  color: var(--near-white);
  padding-left: var(--spacing-large);
  padding-right: var(--spacing-large);
  *zoom: 1;
  box-sizing: border-box;
  display: table;
  width: 100%;
}

.c2 {
  display: table-cell;
  text-align: right;
  margin: var(--spacing-none);
  line-height: 1.5;
}

<header
  className="c0"
>
  <h1
    className="c1"
  >
    Page Title
  </h1>
  <p
    className="c2"
  >
    an Apidura production
  </p>
</header>
`;
